Python/Python.sublime-syntax

```diff
@@ -395,6 +391,15 @@ contexts:
     - match: '@'
       scope: keyword.operator.matrix.python
 
+  allow-unpack-operators:
+    # Match unpacking operators, if present
+    - match: \*
+      scope: keyword.operator.unpacking.python
+    - match: \*\*
+      scope: keyword.operator.unpacking.python
```

Comment:
This pattern will never be matched due to the single `*` pattern coming before it.